packages/earth-map/src/components/collection/collection-editplaces/CollectionEditPlaces.tsx

```diff
@@ -39,9 +46,31 @@ interface IProps {
   reloadCollection: (payload?: any) => void;
 }
 
+const useStyles = makeStyles((theme) => ({
+  root: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    right: 0,
+    bottom: 0,
+    zIndex: 2,
+    display: 'flex',
+    flexDirection: 'column',
+    backgroundColor: theme.palette.background.default,
+  },
+  header: {
+    backgroundColor: theme.palette.grey['600'],
+  },
+  scrollContainer: {
+    flex: '1 1 auto',
+    overflow: 'auto',
+  },
+}));
```

Comment:
This layout is a clear contender for a custom component we should create later on